### DISCUSSION TITLE
DFT+U for molecule

### DISCUSSION THREAD

**Discussion by JunboLu** [pyscf/pyscf#2637] [Q&A]
Hi,
  Is there code of DFT+U for molecule in PySCF?
  With my best regards,
  Junbo

**Comment by simonganne01** [comment]
Maybe you can use the GTO.cell.dimension tag and setting it to 0? I know it is indeed implemented for periodic materials but this way you can maybe extend it to molecules. 

more info here: https://pyscf.org/user/pbc/gto.html#low-dimensional-systems

**Comment by fishjojo** [comment]
There is a differentiable version here https://github.com/fishjojo/pyscfad/blob/main/examples/dft/01-DFT%2BU.py, and you can modify it easily.

**Comment by jeanwsr** [comment]
Implemented in #2835.
